# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/flex/flex.skl.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- flex-2.5.31/flex.skl.old	2004-01-01 20:38:27.000000000 +0100
+++ flex-2.5.31/flex.skl	2004-01-01 20:38:49.000000000 +0100
@@ -478,7 +478,7 @@
 
 m4_ifdef( [[M4_YY_NOT_IN_HEADER]],
 [[
-#define unput(c) yyunput( c, YY_G(yytext_ptr) M4_YY_CALL_LAST_ARG )
+#define unput(c) yyunput( c, YY_G(yytext) M4_YY_CALL_LAST_ARG )
 ]])
 
 /* The following is because we cannot portably get our hands on size_t
